pkg/apis/storage/types.go

```diff
@@ -337,6 +337,41 @@ type CSIDriverSpec struct {
 	//
 	// +optional
 	StorageCapacity *bool
+
+	// ServiceAccountTokens indicates the csi driver needs pods' service account
+	// tokens it is mounting volume for to do necessary authentication. Kubelet
+	// will pass the tokens in VolumeContext in the CSI NodePublishVolume calls.
+	// The CSI driver should parse and validate the following VolumeConext:
+	// "csi.storage.k8s.io/serviceAccount.tokens": {
+	//   'audience-0': {
+	//     'token': token-0,
+	//     'expiry': expiry-0,
+	//   },
+	//   // If audience is not specified, the output key will be ''
+	//   '': {
+	//     'token': token,
+	//     'expiry': expiry,
+	//   },
+	//   ...
+	// }
+	//
+	// Note: Audience in each ServiceAccountToken should be different and at
+	// most one token can ignore Audience.
+	//
+	// +optional
```

Comment:
Add a comment that this field is alpha-level and requires the feature gate to be set. Ditto everywhere

Reply:
done